lib/CfTemplateGenerators/Lambda.js

```diff
@@ -13,10 +13,12 @@ function buildUpdatePolicy ({ codeDeployApp, deploymentGroup, afterHook, beforeH
   return omitEmpty({ UpdatePolicy: updatePolicy })
 }
 
-function buildAlias ({ alias, functionName, functionVersion, trafficShiftingSettings }) {
+function buildAlias ({ alias, functionName, functionVersion, trafficShiftingSettings, targetAliasTemplate }) {
   const lambdaAlias = {
+    ...targetAliasTemplate,
```

Comment:
I'm not too sure _what_ we're expecting to be in the `targetAliasTemplate` object but I wanted to check how the JS spread operator handled collisions and it looks like any non-unique keys with those defined below will be overwritten, is that the behaviour we expect? What unique keys are we expecting in the `targetAliasTemplate` ?

Reply:
The keys we care about are applied on top of the spread as defined in the subsequent lines, as per the following rule in the language reference:

> When one object is spread into another object, or when multiple objects are spread into one object, and properties with identical names are encountered, the property takes the last value assigned while remaining in the position it was originally set.

See [reference in question](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_object_literals).